lib/es/esClient.js

```diff
@@ -117,6 +117,24 @@ async function deleteIndex(index) {
   }
 }
 
+async function removeOrphanedIndexes(alias) {
+  const allIndexes = await client.indices.getAlias();
+  const orphans = Object.keys(allIndexes).filter(key => key.startsWith(`${alias}_`) && !allIndexes[key].aliases[alias]);
+  /* eslint-disable */
```

Comment:
Does this need to be here or could specific ignores be used?
If it is required can eslint be enabled after the disabled bit has ended?

Reply:
I'll make it explicit